### PR TITLE
Fabricator Efficiency Removal

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Gosh darn it, @KorPhaeron if you're going to remove this, remove it from everything. :stuck_out_tongue: 

For the record, I do not support this PR, but if R&D is going to be re-balanced (or materials costs/sheet amount/material amount per sheet), then cost-decreases should be properly stripped from everything, and not just three of the four major R&D-related machines.
- Removes design cost reduction from upgrading mech fab parts
- Removes design cost reduction from increasing materials tech level
- Removes design speed increase from increasing programming tech level
  - Speed is still increased by upgrading the parts of the mech fab

:cl: Fox McCloud
remove: Removes mech fab tech level speed increased and design cost reduction
remove: Removes mech fab parts design cost reduction
/:cl:

### CODE REVIEW

**Comment by RandomMarine** [comment]
Better idea.

1. Bring back protolathe upgrades.
2. Remove ORM upgrades, the thing that caused this big balance mess.

Badaboom.

**Comment by Bawhoppen** [comment]
:+1: just remove everything for now and then we'll fix everything else later

File: code/game/mecha/mech_fabricator.dm [hunk 1]
```diff
@@ -10,9 +10,6 @@
 	active_power_usage = 5000
 	req_access = list(access_robotics)
 	var/time_coeff = 1
-	var/resource_coeff = 1
-	var/time_coeff_tech = 1
```
Comment (KorPhaeron) [inline code/game/mecha/mech_fabricator.dm]: Why is this one being removed?

**Comment by swankcookie** [comment]
delete science

File: code/game/mecha/mech_fabricator.dm (hunk 1 above)
Comment (Fox-McCloud) [inline code/game/mecha/mech_fabricator.dm]: Sure thing.

So mech fabs have a special thing where increasing the materials levels decreases the cost of designs.

They also have a thing where increasing the programming level makes things produced faster as well.

I removed both; I didn't remove faster production by upgrading the *parts* only by increasing the programming level.

It didn't make much sense to have speed increased by two independent factors; figured it best for it to be handled *purely* by part upgrades.

**Comment by Robustin** [comment]
I'd rather have RND mass-production come from an effective mining team than science being a self-sustaining meme machine. However, once you get out of RND, I don't see why other areas can't benefit from efficiency. I haven't made a mech lately but the old efficiencies were pretty reasonable, it made mechs affordable but not spammable.

**Comment by Fox-McCloud** [comment]
mech fabs, for all intents and purposes, are part of R&D and visa versa.

Robotics can do R&D and so can R&D itself---it's not hard to get spare boards from tech storage for robotics...and R&D can easily build a mech fab at 5 minutes in....andd robotics has access to the R&D console anyway.

IMO, the current big problem robotics has is they only have frigging 20 sheets of glass but something like 300 sheets of metal.

**Comment by coiax** [comment]
> IMO, the current big problem robotics has is they only have frigging 20 sheets of glass but something like 300 sheets of metal.

I don't know, that seems about right. Building a new cyborg is mostly just a huge metal investment.

**Comment by ChangelingRain** [comment]
I might get yelled at for this but R&D is dumb so oh well.

**Comment by GunHog** [comment]
Yet ANOTHER middle finger to Science players.

**Comment by Core0verload** [comment]
wait what

**Pull request merged**